src/main/java/org/traccar/api/SimpleObjectResource.java

```diff
@@ -52,8 +53,13 @@ public Collection<T> get(
             }
             conditions.add(new Condition.Permission(User.class, userId, baseClass));
         }
-
-        return storage.getObjects(baseClass, new Request(new Columns.All(), Condition.merge(conditions)));
+        Order order = null;
+        if(hasField("name")) {
```

Comment:
Maybe instead of reflection we can explicitly pass ordering field in the constructor? Also I'm not sure we have name or description for everything. So probably need to support `null` option as well.

Reply:
I really hadn't thought of that; I redid it.